stage #2.5/module #4 REST Basics - Spring Web/Spring Rest/spring-rest-controller-content.md

```diff
@@ -0,0 +1,86 @@
+## Rest Controller
+
+Let's consider how to build REST controllers using Spring Web framework.
+Spring REST controllers are annotated with **@RestController**. The annotation includes the **@RestController** and **@ResponseBody** annotations, and as a result, simplifies the controller implementation:
+```Java
+@RestController
+@RequestMapping(value = "/books", consumes = {"application/JSON"}, produces = {"application/JSON", "application/XML"})
+public class BookController {
+    @Autowired
+    private BookService bookService;
+
+    @Autowired
+    private Convertor<BookDTO, Book> convertor;
```

Comment:
I think it's not good idea convert something at the Controller level. Service level is present for it. It can be more flexible to work with DTO at the service level.